scripts/caravan/caravan.lua

```diff
@@ -1143,6 +1144,16 @@ end
 py.register_on_nth_tick(60, "update-caravans", "pyal", function()
     local guis_to_update = {}
 
+    for _, player in pairs(game.players) do
```

Comment:
For multiplayer compatibility, this should be

    for _, player in pairs(game.connected_players) do